diesel/src/types/mod.rs

```diff
@@ -174,6 +174,8 @@ impl SingleValue for Numeric {}
 /// On all backends strings must be valid UTF-8.
 /// On PostgreSQL strings must not include nul bytes.
 ///
+/// On Mysql, it is also aliased by Tinytext, Mediumtext, Longtext, Char and Varchar.
```

Comment:
Just cosmetics, but can you change it to "MySQL" and put the type names in back ticks (like <code>\`this\`</code>).